content/docs/uncontrolled-components.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: uncontrolled-components
-title: Uncontrolled Components
+title: 不可控制元件
```

Comment:
```suggestion
title: Uncontrolled Component
```

We reserve the `uncontrolled` term in the previous translation.

See:

- https://github.com/reactjs/zh-hant.reactjs.org/blob/master/content/docs/forms.md
- https://github.com/reactjs/zh-hant.reactjs.org/blob/master/content/docs/reference-dom-elements.md